### DISCUSSION TITLE
Job scheduling is where?

### DISCUSSION THREAD

**Discussion by ssid334** [nautobot/nautobot#3468] [General]
Hey there,

I'm trying to learn nautobot. I did create simple job already and now looking at the feature to schedule it.
The doc says it is Jobs->Jobs but I'm looking at my local installation (I do have celery worker running) and even
at the online demo job: https://demo.nautobot.com/extras/jobs/plugins-nautobot_firewall_models-jobs-runcapircajob/run/

and... I have no idea where is the scheduler option. Please help?

Roman

**Comment by jdrew82** [comment]
The scheduling of a Job is done through the Job form itself. It will appear as a smaller box on the form called `Job Execution` at the bottom with a dropdown where you select the timeframe and schedule it.

<img width="919" alt="Screenshot 2023-03-21 at 3 37 59 PM" src="https://user-images.githubusercontent.com/2396364/226734997-c10fd59f-31b2-4ee4-9c3e-dc04bf58edd5.png">

However, in order for that functionality to work you'll need to ensure that you have the `Has sensitive variables` property in the Job settings toggled off. Without this setting toggled off the Job can't save the form settings for the Job that is required for it to be scheduled.

**Reply by bryanculver** [reply]
![oh hey](https://media.giphy.com/media/l36kU80xPf0ojG0Erg/giphy.gif)

**Reply by ssid334** [reply]
Yeah, it works. I had to enable this "Has sensitive variables". Thank you for your help.
I think this could be documented better or the "Job execution" div should be grayed-out with the information that it is disabled due to the sensitive variable parameter being enabled?

**Comment by bryanculver** [comment]
**EDIT:** ☝️ What @jdrew82 said
